mime.types.php

```diff
@@ -33,6 +33,8 @@
     array (
       0 => 'application/x-gzip',
       1 => 'application/zip',
+      2 => 'application/x-zip-compressed',
+      3 => 'application/x-zip',
```

Comment:
On `README`, it has this message:

> This package uses httpd's mime.types to generate a mapping of file extension to MIME type and the other way around.

It seems that these zip mime types are not mentioned on this [mapping list](https://svn.apache.org/repos/asf/httpd/httpd/trunk/docs/conf/mime.types).